clustergroup/values.schema.json

```diff
@@ -266,6 +266,13 @@
             "type": "string"
           }
         },
+        "operatorgroupExcludeTargetNS": {
+          "type": "array",
+          "description": "List of namespaces to exclude the target namespace.",
```

Comment:
Maybe call it `Specify the list of namespaces where the target namespace field in the corresponding operatorgroup object should be excluded`

Reply:
Done.